.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
```

Optional, but to keep up with new node releases, it might be better to use:

```suggestion
        node-version: [lts/-2, lts/-1, lts/*] 
```

---

Oh yeah, this is great. Again, got this from the GitHub starter workflow. Should have thought about this.